cpp/daal/include/services/env_detect.h

```diff
@@ -91,7 +97,12 @@ class DAAL_EXPORT Environment : public Base
     enum CpuTypeEnable
     {
         cpu_default = 0, /*!< Default processor type */
-        avx512      = 2  /*!< Intel(R) Xeon(R) processors based on Intel(R) Advanced Vector Extensions 512 (Intel(R) AVX-512) \DAAL_DEPRECATED */
+        
+        #ifdef __ARM_ARCH
+            sve         = 2, /*!< ARM processors based on Arm's Scalable Vector Extension (SVE) */
```

Comment:
Should this have a value of 2? As this is the only CPU arch for Arm, this should be 1?

Reply:
assumption was neon would be 1 in the future.